### ISSUE TITLE
accepts strings for input to robot query and print results to screen

### ISSUE THREAD

**Issue by wdduncan** [ontodev/robot#695]
It would be helpful if `robot query` accepted strings as the input and allowed the results to be printed to stdout. E.g., something like:
```
robot query -i foo.owl \
-q 'select * where {?s ?p ?o} limit 10' \
-stdout true
```
Currently, you can do some hacks to get similar behavior. E.g.,
```
“select * where {?s ?p ?o} limit 10” >> s.sparql && robot query -i o.owl —query s.sparql -o out.owl && rm s.sparql
```
```
cat <<< 'select * where {?s ?p ?o} limit 10' | robot query -i o.owl -q /dev/stdin /dev/stdout
```

cc @matentzn

**Comment by jamesaoverton** [comment]
I agree that this would be convenient, and I'd be happy to accept a PR.

If somebody plans to work on it, before you begin let's nail down the details of the option names, if/how you want to handle STDIN and multiple queries, and interactions with the rest of `query`.

I do want to point out that running a bunch of one-off SPARQL queries will be slow unless the TDB options are used. Without TDB `query` is expensive:

- load the ontology with OWLAPI
- render the ontology triples to into an in-memory Jena graph
- run the query
- throw away the in-memory graph
- throw away the in-memory ontology

With TDB you save time but pay for it with a bit of storage.
